sdk/monitor/azingest/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,76 @@
+# Troubleshooting Azure Monitor Ingestion module
+
+This troubleshooting guide contains instructions to diagnose frequently encountered issues while using the Azure Monitor Ingestion module for Go.
+
+## Table of contents
+
+* [General troubleshooting](#general-troubleshooting)
+    * [Error Handling](#error-handling)
+    * [Logging](#logging)
+* [Troubleshooting logs ingestion](#troubleshooting-logs-ingestion)
+    * [Troubleshooting authorization errors](#troubleshooting-authorization-errors)
+    * [Troubleshooting missing logs](#troubleshooting-missing-logs)
+    * [Troubleshooting slow logs upload](#troubleshooting-slow-logs-upload)
+
+## General troubleshooting
+
+### Error Handling
```

Comment:
```suggestion
### Error handling
```

Reply:
fixed in [d087998](https://github.com/Azure/azure-sdk-for-go/pull/21459/commits/d08799861214cf1cb2e7fdcf8243bd20210d2fef)